resources/lang/en/events/injury/desert.json

```
@@ -3259,6 +3275,23 @@
             }
         ]
     },
+    {
+        "event_id": "dst_injury_scrapessmallcuts_appmultitrait1",
+        "location": ["desert"],
+        "season": ["any"],
+        "weight": 20,
+        "event_text": "m_c dashes off from {PRONOUN/m_c/poss} mentor and tumbles face first into a cactus, thankfully the medicine cat is able to remove the spines with little difficulty.",
```

```suggestion
        "event_text": "m_c dashes off from {PRONOUN/m_c/poss} mentor and tumbles face first into a cactus! Thankfully, the medicine cat is able to remove the spines with little difficulty.",
```

Though I'm also wondering how this is prevented from rolling for cats with no mentor, or adolescents that graduated early?

---

It does not infact, changing that event up a bit to fix this issue
Instead im having the cat's mentor remove the spines to help explain the minor injuries given, adding in a reationship_status constraint to m_c of app/mentor, and added r_c in to allow that to work